designer/server/src/config.ts

```diff
@@ -37,14 +38,18 @@ export interface Config {
 const schema = joi.object<Config>({
   port: joi.number().default(3000),
   env: joi
+    .string()
+    .valid('development', 'test', 'perf test', 'production')
```

Comment:
I'm still thinking we should match these names to how they're used in URLs

We might want to use them to replace wildcards in URLs in future

```suggestion
    .valid('dev', 'test', 'perf-test', 'prod')
```

Reply:
I will let @alexluckett decide that I think! 

This is for non production envs, so I don't have strong opinion

Reply:
I'll ask on Slack

What shall we use when running locally?

Because the "dev" environment isn't local, e.g.

http://forms-designer.dev.cdp-int.defra.cloud
http://forms-runner.dev.cdp-int.defra.cloud

We'll need maybe `local` or `localhost` as the default